src/Sylius/Bundle/AdminBundle/Resources/config/grids/customer_order.yml

```diff
@@ -5,7 +5,7 @@ sylius_grid:
             driver:
                 options:
                     repository:
-                        method: createByCustomerIdQueryBuilder
+                        method: createByCustomerIdCriteriaAwareQueryBuilder
                         arguments:
                             customerId: $id
```

Comment:
Hey, I have a question about this code. This shouldn't work. The fist argument of the function is the criteria but in here you specify the customer id first. See:

[Grid bundle driver.](https://github.com/Sylius/SyliusGridBundle/blob/61970651ba4cf6547cb6b0d3220d7c9891f5e70d/src/Bundle/Doctrine/ORM/Driver.php#L53)